XRTK-Core/Packages/com.xrtk.core/Editor/PackageInstaller.cs

```diff
@@ -43,9 +43,13 @@ public static bool TryInstallAssets(string sourcePath, string destinationPath, b
                 EditorApplication.delayCall += () => AddConfigurations(installedAssets);
                 return true;
             }
+            else
+            {
+                Directory.CreateDirectory(Path.GetFullPath(destinationPath));
```

Comment:
I think we should always do this at the top of the method if the directory doesn't exist

Reply:
It does?  The first line checks if the directory exists.  This is then the "Else" on that check?

Reply:
Ah you're right. we have a different path if it exists.